integration-tests/go.mod

```diff
@@ -4,16 +4,22 @@ go 1.19
 
 require (
 	github.com/dontpanicdao/caigo v0.4.0
-	github.com/go-resty/resty/v2 v2.7.0
-	github.com/onsi/ginkgo/v2 v2.4.0
-	github.com/onsi/gomega v1.23.0
 	github.com/rs/zerolog v1.28.0
 	github.com/satori/go.uuid v1.2.0
-	github.com/smartcontractkit/chainlink v1.9.0
+	github.com/smartcontractkit/chainlink v1.10.0
+	github.com/smartcontractkit/chainlink-env v0.3.1
 	github.com/smartcontractkit/chainlink-starknet/ops v0.0.0-20221206105830-e9411b085b4a
 	github.com/smartcontractkit/chainlink-starknet/relayer v0.0.0-20221103134312-d40949720ad3
-	github.com/smartcontractkit/chainlink-testing-framework v1.7.9-0.20221103145957-d40b562b818b
-	github.com/smartcontractkit/chainlink/integration-tests v0.0.0-20221102233518-1e61a9b970a2
+	github.com/smartcontractkit/chainlink-testing-framework v1.9.2
+	github.com/smartcontractkit/chainlink/integration-tests v0.0.0-20230120175917-456bc0c86b97
+)
+
+require (
+	github.com/smartcontractkit/ocr2keepers v0.6.6 // indirect
+	github.com/stretchr/objx v0.5.0 // indirect
+	github.com/umbracle/ethgo v0.1.3 // indirect
+	github.com/umbracle/fastrlp v0.0.0-20220527094140-59d5dd30e722 // indirect
+	github.com/valyala/fastjson v1.4.1 // indirect
 )
 
 require (
```

Comment:
can we combine the two `// indirect` sections to keep it cleaner? (not sure why go decides to split things)